models.py

```diff
@@ -36,9 +36,11 @@ class User(Base, UserMixin):
     password = Column(String(256))
     active = Column(Boolean())
     confirmed_at = Column(DateTime())
+    social_id = Column(String(64), nullable=True, unique=True)
```

Comment:
This will require a migration. You should create one with Alembic and put it under `alembic/versions`.